content/tools/ruby.md

```diff
@@ -66,7 +66,7 @@ while running `rake cucumber:ok` will run the Steps within all Scenarios other t
 
 ### Using Profiles in Rake Tasks
 
-For complex Feature runs that are tested often, it is nice to save the command line arguments as [Cucumber profiles](/cucumber/configuration/#defining-profiles).
+For complex Feature runs that are tested often, it is nice to save the command line arguments as [Cucumber profiles](/cucumber/configuration#profiles).
```

Comment:
Sidenote: afaik the links were fine, but they still seem to work like this, so ok.

Reply:
It complained.

Reply:
Hmm, must be new. Glad it works :)